### PR TITLE
sky-view:Interactive tool to show location of gaia sources on a TPF.

### PR DESCRIPTION
**Pull request by gully**
Sky view is a tool that reveals the identity and number count of Gaia point sources in your TPF, like so:
gaia_tpf_view

### CODE REVIEW

**Comment by gully** [comment]
Ok, the Skyview now works:  Within a Jupyter Notebook you can do:

```python
tpf.show_gaia()
```

And an interactive plot pops up with hover-over tool tips for each Gaia source, as shown in the gif animation above.  I have also added a screen stretch slider, with the same behavior as the `interact` function.  There remain some tasks for incremental improvement:

- [ ] Currently the radius for querying Gaia is *hardcoded* to 90 arcseconds.  Instead, determine the size based on the TPF.
- [ ] Refactor the variable names.
- [ ] Both this widget and `interact` repeat the same screen-stretch code, which violates "don't repeat yourself".  Consider making a tpf.screen_stretch attribute that returns a "sane" screen stretch based on the behavior implemented here.
- [ ] Replace the custom Gaia querying code with a method tpf.query_gaia( ), that returns a list of sources and their metadata.

**Comment by christinahedges** [comment]
@gully this is SO good I need this in my life. I've just made a few small changes to bring this tool up to date with current lightkurve. Here are the changes I've made:

1. The widget is now in the `interact` module
2. After some testing, the widget breaks if you look at a large tpf (e.g. C9) which is where this widget really shines! I've tweaked the astroquery portion to always give the right answer
3. I added a `magnitude_limit` keyword so that we're not inundated with points in C9, it's 18 by default.
4. I added some tweaks to make the xrange and yrange always the full TPF size
5. I added the proper motion from Gaia in. This might help if we're skyviewing a Kepler tpf as there can be up to 10 years of proper motion...! Otherwise this is a tiny tweak.

I was thinking about the name...
How about `tpf.skymap` for the name?

Also I'm taking WIP off this because this is a full blown tool and I need it.

**Comment by gully** [comment]
Needs to be rebased on master since the updates to `Save Lightcurve` in interact

**Comment by gully** [comment]
20190131_lightkurve_skyview

**Comment by gully** [comment]
This PR is ready for review.  See the demo above.
@barentsen 
@christinahedges

File: lightkurve/interact.py [hunk 1]
```diff
@@ -194,7 +200,58 @@ def make_lightcurve_figure_elements(lc, lc_source):
...
+    # Use pixel scale for query size
+    pix_scale = 4.0 # arcseconds / pixel for Kepler, default
+    if tpf.mission == 'TESS':
+        pix_scale = 21.0
+    # We are querying with a diameter as the radius, overfilling by 2x.
+    result = Vizier.query_region(c1, catalog=["I/345/gaia2"], radius=Angle(np.max(tpf.shape[1:]) * pix_scale, "arcsec"))
+    if result is None:
+        raise ValueError('No targets found in region.')
+    result = result["I/345/gaia2"].to_pandas()
+    result = result[result.Gmag < magnitude_limit]
```
[the remark this answers is not in the document]
Comment (gully) [reply lightkurve/interact.py]: Here is an update notifying the user of both a None result or an empty list:

```python
no_targets_found_message = LightkurveWarning('No sources found in the query region.')
    if result is None:
        raise no_targets_found_message
    elif len(result) == 0:
        raise no_targets_found_message
```

**Comment by barentsen** [comment]
@gully Thank you for working on this!

I went ahead and added `notebook_url` as an optional parameter to `interact_sky()`, to make it consistent with `interact()` and `interact_bls()`.

One remaining conundrum is that this example only shows the pixels and does not show the Gaia catalog sources overlaid, even though TessCut TPFs do have a WCS etc:
```python
import lightkurve as lk
tpf = lk.search_tesscut("Pi Men", sector=1).download(cutout_size=13)
tpf.interact_sky()
```

**Comment by barentsen** [comment]
I believe #434 will resolve the issue noted above.

**Comment by barentsen** [comment]
#434 has been merged and I confirmed that `interact_sky()` now works on TESS cut-outs.  Merging this once the tests pass!

**Comment by megbedell** [comment]
gully this is the best thing ever!!!!

**Comment by keatonb** [comment]
Love this @gully.  However, the information on mouseover doesn't seem correct.  Notice in your animated demonstration that all the source IDs end with zeros, while the actual Gaia source IDs don't, making it difficult to use these IDs to look up more Gaia catalog info.  Also the displayed RAs,Decs don't match the Gaia catalog info (are these adjusted to align with the TESS pixels?).

**Comment by gully** [comment]
Hey @keatonb, good eye!  Indeed the truncated source IDs is a known bug that should be fixable-- I will investigate.  The RA and Decs are different---I suspect--- because we adjust for the proper motion of the source **evaluated at the epoch of Kepler/TESS observation**.  This change should be minuscule for TESS and Kepler, with their large-on-the-sky pixels.  But it's conceivable that very-high proper motion sources would exhibit measurable differences.  I will sanity check, to be sure.

**Pull request merged**